components/sup/src/event.rs

```diff
@@ -233,14 +235,17 @@ pub fn health_check(metadata: ServiceMetadata,
         let StandardStreams { stdout, stderr } =
             maybe_process_output.map(ProcessOutput::standard_streams)
                                 .unwrap_or_default();
+        let interval: u64 = health_check_interval.into();
```

Comment:
It is generally preferred to leave off explicit type definitions in Rust. The compiler should be able to infer what type we want to convert into here.

Reply:
Done!

Reply:
Done!